css/asciidoctor.css

```diff
@@ -1923,3 +1923,7 @@ p {
 .print-only {
   display: none !important
 }
+
+.stretch {
+  width: 100%
```

Comment:
Good catch!
We should sync the stylesheet with https://github.com/asciidoctor/asciidoctor/blob/master/data/stylesheets/asciidoctor-default.css

Not necessarily blindly overwriting this file but cherry-picking rules.